node/packages/parser/src/ethereum/index.ts

```diff
@@ -21,15 +22,18 @@ export interface TransactionParserArgs {
   rpcUrl: string
 }
 
-export type Parser = thor.Parser | uniV2.Parser | zrx.Parser
+export interface GenericParser {
+  parse: (tx: Tx) => Promise<Partial<TxSpecific<ParseTx>> | undefined>
```

Comment:
Just noticed we are redundantly using `Partial` here and in the zrx parser function since `TxSpecific` is already a partial type

Reply:
Good catch, resolved in https://github.com/shapeshift/unchained/pull/267/commits/5606fc19cb7ae325866e20a8fa0d124f4330d5c0.